content/pages/sphinx.html

```diff
@@ -0,0 +1,243 @@
+<html>
+  <head>
+    <title>Easily build and deploy your Sphinx docs</title>
+    <meta name="template" content="page" />
+    <meta name="slug" content="landing/sphinx" />
+    <meta name="status" content="hidden" />
+    <meta name="description" content="Learn more about Sphinx & Read the Docs hosting" />
+  </head>
+  <body>
+
+{#
+  View this page at http://localhost:8080/landing/sphinx/index.html
+#}
+
+{% import "partials/basic.html" as basic %}
+{% import "partials/marketing.html" as marketing %}
+{% import "partials/technical.html" as technical %}
+
+<section>
+  {% call basic.header_image(
+    header="Quickly deploy your Sphinx docs across multiple versions",
+    image="https://cdn.dribbble.com/users/876899/screenshots/2141696/readthedocs-thumb.png",
+    image_alt="Marketing introduction header",
+    vertical_align="middle")
+  %}
+    <p>
+        Beautiful docs for your team are just a <code>git push</code> away.
+    </p>
+
+    <p class="ui basic padded vertical segment">
+      <a href="#" class="ui large stackable primary button">
```

Comment:
```suggestion
      <a href="https://readthedocs.org/accounts/signup" class="ui large stackable primary button">
```

Should we link people to .org or .com here now? However, I suppose this will show the same modal than from the homepage:

![Screenshot_2022-08-25_18-21-25](https://user-images.githubusercontent.com/244656/186718189-51f91c9c-9ac3-4c6c-afb2-35aacaad7c39.png)

Reply:
We already added the modal on these buttons.